get-latest.sh

```diff
@@ -36,11 +36,32 @@ if [ $? -ne 0 ]; then
 	exit 1;
 fi
 
-# Get a list of version numbers that are available
-cat /tmp/tc${tomcat_major_version}-index.html | fgrep '[DIR]' | grep -E "v${tomcat_major_version}.${tomcat_minor_version}.[0-9]+/" | grep -Eo '<a[ \t]+href[ \t]*=[ \t]*"[^"]*"' | grep -Eo "v${tomcat_major_version}\.${tomcat_minor_version}\.[^\"]*" | sed 's|/$||' > /tmp/tc${tomcat_major_version}-versions.txt
```

Comment:
The values gotten from _cat_ and friends were redirected out to a file. Maybe that is what is needed for _version_str_ to start working.